### PR TITLE
Add PowerShell

### PR DESCRIPTION
**Pull request by TravisEz13**
## Project URL
https://github.com/powershell/powershell

## Category
Utilities

## Description
PowerShell is a shell.
> PowerShell Core is a cross-platform (Windows, Linux, and macOS) automation and configuration tool/framework that works well with your existing tools and is optimized for dealing with structured data (e.g. JSON, CSV, XML, etc.), REST APIs, and object models. It includes a command-line shell, an associated scripting language and a framework for processing cmdlets. 
## Checklist

- [x] Edit [applications.json](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/applications.json) instead of [README.md](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/README.md).
- [x] Only one project/change is in this pull request
- [ ] Screenshots(s) added if any
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by TravisEz13** [comment]
Please don't update `https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-macos?view=powershell-6` to `https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-macos?view=powershell-6`.

This is not a static redirect.  It will redirect based on the user language.

File: applications.json [hunk 1]
```diff
@@ -7255,6 +7255,20 @@
...
+            "categories": [
+                "utilities"
+            ],
+            "repo_url": "https://github.com/powershell/powershell",
+            "title": "PowerShell",
+            "icon_url": "https://raw.githubusercontent.com/PowerShell/PowerShell/master/assets/Powershell_black_64.png",
+            "screenshots": [],
+            "official_site": "https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-macos?view=powershell-6",
+            "languages": [
+                "c#"
```
Comment (serhii-londar) [inline applications.json]: Please change from "c#" to "c_sharp".

Comment (TravisEz13) [reply applications.json]:
```suggestion
                "c_sharp"
```

**Comment by serhii-londar** [comment]
@TravisEz13 Thank you! 🚀

**Pull request merged**